src/redux/books/books.js

```diff
@@ -0,0 +1,45 @@
+// actions
+let nextBookId = 2;
+export const addBook = (title, author, genre) => {
+  nextBookId += 1;
+  return (
+    {
+      type: 'ADD_BOOK',
+      id: toString(nextBookId),
+      title,
+      author,
+      genre,
+    });
+};
+
+export const removeBook = () => ({
+  type: 'REMOVE_BOOK',
+});
```

Comment:
- [ ] There are actions creators but you should create the action as in [ducks pattern](https://github.com/erikras/ducks-modular-redux) I'll let you the link to take a look in order to complete the requirements